src/LibraryManager/Providers/Cdnjs/CdnjsProvider.cs

```diff
@@ -237,35 +239,21 @@ public string GetSuggestedDestination(ILibrary library)
 
         private async Task<Stream> GetStreamAsync(ILibraryInstallationState state, string sourceFile, CancellationToken cancellationToken)
         {
-            string name = GetLibraryName(state.LibraryId);
-            string version = GetLibraryVersion(state.LibraryId);
+            (string name, string version) = LibraryNamingScheme.Instance.GetLibraryNameAndVersion(state.LibraryId);
```

Comment:
You shouldn't have to do this. 
The `ILibraryInstallationState` has the name and version properties already.